src/data/list/basic.lean

```diff
@@ -1222,6 +1222,28 @@ ext $ λn, if h₁ : n < length l₁
   then by rw [nth_le_nth, nth_le_nth, h n h₁ (by rwa [← hl])]
   else let h₁ := le_of_not_gt h₁ in by { rw [nth_len_le h₁, nth_len_le], rwa [←hl], }
 
+lemma injective_append_left (l : list α) : function.injective (list.append l) :=
```

Comment:
```suggestion
lemma injective_append_left (l : list α) : function.injective ((++) l) :=
```
since I assume `++` is the preferred spelling of `list.append`.